server/resolvers/TestPlanReportOperations/updateStatusResolver.js

```diff
@@ -51,49 +57,31 @@ const updateStatusResolver = async (
             }
         });
 
-        const candidateStatusReachedAt = new Date();
-        updateParams = {
-            ...updateParams,
-            candidateStatusReachedAt,
-            metrics: { ...testPlanReport.metrics, ...metrics },
-            recommendedStatusTargetDate: recommendedStatusTargetDateResolver({
-                candidateStatusReachedAt
-            })
-        };
-    } else if (status === 'FINALIZED') {
-        const conflicts = await conflictsResolver(testPlanReport);
-        if (conflicts.length > 0) {
-            throw new Error(
-                'Cannot finalize test plan report due to conflicts'
-            );
+        if (status === 'IN_REVIEW') {
+            updateParams = {
+                ...updateParams,
+                metrics: { ...testPlanReport.metrics, ...metrics }
+            };
+        } else if (status === 'CANDIDATE') {
+            const candidateStatusReachedAt = new Date();
+            updateParams = {
+                ...updateParams,
+                candidateStatusReachedAt,
+                metrics: { ...testPlanReport.metrics, ...metrics },
+                recommendedStatusTargetDate: recommendedStatusTargetDateResolver(
+                    {
+                        candidateStatusReachedAt
+                    }
+                )
```

Comment:
`vendorReviewStatus: 'READY'` should go here

Reply:
The `vendorReviewStatus` gets set when the report goes to the Candidate phase. Once it's been recommended, the `vendorReviewStatus` should be approved.

Reply:
```suggestion
                ),
                vendorReviewStatus: 'READY'
```